test/client/components/Price.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Price from 'client/components/Price/Price';
+import renderer from 'react-test-renderer';
+
+test('With Enzyme, Price component will show 3 checks', () => {
+  const price = '3,4';
+  const wrapper = shallow(
+    <Price price={price} />,
+  );
+  let total = 0;
+  const b = wrapper.find({ type: 'checkbox' });
+  b.map((item) => {
```

Comment:
why `.map` ?

Reply:
consider other methods on checking the number of checked checkboxes instead of looping through all of them and using a counter.